00-HTML-CSS-basics/ex1.V.html

```diff
@@ -0,0 +1,23 @@
+
+<html>
+<head>
```

Comment:
Indentation here. Should be one more since it's inside `<html>`. Same for everything below.